### PR TITLE
ScreenShots Added and .gitignore updated.

### PR DESCRIPTION
**Pull request by RishKhanna**
-> Screenshots of the gameplay have been added to the README.md.
-> "__pycache__" has been added to ".gitignore".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ The main idea behind the game is to make it an expansive user-modded game. Thus,
 Some of the commands that have been thought of are - protect, reverse, slow, save, wildcard etc.
 The player will not be knowing which all commands exist in the game, but rather be expected to try out the different words that can be formed with the available keys, and see what happens and works.
 
+## Game Play
+<img src="./assets/images/Home_Page.png">
```
Comment (LakshyAAAgrawal) [inline README.md]: instead of the HTML img tag, you could make use of  image tag in markdown.

**Comment by LakshyAAAgrawal** [comment]
Thank you so much @KyaBacchaHai for the PR. I have left a comment, you could resolve that and I would be happy to merge the PR.

**Comment by RishKhanna** [comment]
I have changed the HTML tags to Markdown tags.

**Comment by LakshyAAAgrawal** [comment]
This is a great contribution. Thank you very much @KyaBacchaHai

**Pull request merged**